contracts/governance/StakingRewards/StakingRewardsStorage.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.5.17;
+
+import "../../interfaces/IERC20.sol";
+import "../Staking/IStaking.sol";
+import "../../openzeppelin/Ownable.sol";
+
+/**
+ * @title Staking Rewards Storage Contract.
+ * @notice Just the storage part of staking rewards contract, no functions,
+ * only constant, variables and required structures (mappings).
+ * Used by StackingRewardsProxy.
+ *
+ * What is SOV staking rewards - SIP-0024?
+ * The purpose of the SOV staking rewards - SIP-0024 is to reward,
+ * "marginal stakers" (ie, stakers by choice, not currently vesting) with liquid SOV
+ * at the beginning of each new staking interval.
+ * */
+contract StakingRewardsStorage is Ownable {
+	/// @notice The SOV token contract.
+	IERC20 public SOV;
+
+	///@notice the staking proxy contract address
+	IStaking public staking;
+
+	/// @notice 2 weeks in seconds.
+	uint256 constant TWO_WEEKS = 1209600;
+
+	/// @notice Annual Base Rate - it is the maximum interest rate(APY)
+	uint256 constant BASE_RATE = 2975;
+
+	/// @notice DIVISOR is set as 2600000 = 26 (num periods per year) * 10 (max voting weight) * 10000 (2975 -> 0.2975)
```

Comment:
Copy from the documentation:

```
Let m be the maximum staking period (1092 days) , x be the number of days passed since the beginning of the period, and V be the maximum voting weight (currently 9). The voting weight at a given time x can be described as:

f(x)=V*(m^2-x^2)/m^2+1

+1 is added in the end to shift the weights to lie in [1, V+1] instead of [0, V].
```

As you can see, we use [1, V+1], which is [1, 10] if V is 9. This means that the maximum weight a stake receives is 10, even though V is set to 9.